### ISSUE TITLE
Gamescope doesn't move the mouse cursor at the same speed the host uses

### ISSUE THREAD

**Issue by orowith2os** [Plagman/gamescope#542]
I have a mouse that can change the DPI without having any software on the host (Razer Mamba Wireless). I have it set to a specific DPI on the mouse, rather than changing it on the host system, so the mouse cursor speed is the same across all my systems.
When I'm using gamescope, I have to change my mouse speed on the mouse again to get it to work properly inside of the gamescope window.

POSSIBLE FIX:
Would it be possible to fix this issue by reading the host mouse cursor settings and using those?

**Comment by ilievi187** [comment]
yeah i have same problem "Razer DeathAdder", Arch , Gamescope -latest

**Comment by UltraBlackLinux** [comment]
https://github.com/Plagman/gamescope/issues/196
